chroma-manager/tests/utils/remote_firewall_control.py

```diff
@@ -187,19 +187,28 @@ def process_rules(self):
         result = self.remote_access_func(self.address, self.firewall_list_cmd)
 
         if result.rc != 0:
-            from chroma_common.lib.shell import Shell
-            raise RuntimeError('''process_rules(): remote shell command failed unexpectedly (%s), is firewall-cmd running? (%s) (%s)
-systemctl status firewalld:
-%s
-
-systemctl status polkit:
-%s
-
-journalctl -n 100:
-%s''' % (result.rc, result.stdout, result.stderr,
-         Shell.run(['systemctl', 'status', 'firewalld']).stdout,
-         Shell.run(['systemctl', 'status', 'polkit']).stdout,
-         Shell.run(['journalctl', '-n', '100']).stdout))
+            if result.rc == 254 and ' is not registered' in result.stderr:
+                from time import sleep
+                # restart polkit to attempt to address https://bugzilla.redhat.com/show_bug.cgi?id=1221483
+                self.remote_access_func(self.address, 'systemctl restart polkit')
```

Comment:
this patch will test the hypothesis that restarting polkit addresses the issue and if not will report information on the failure (on the correct host). it's not something we have to land